spec/factories/form.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :form do
+    form_master_num { 'oh_jfs_04221' }
```

Comment:
This shouldn't be a static value. Writing tests against it will be hard because we'll only be able to create the one form event relationship. For now, you could just use `Faker` to generate any string here.